sound/soc/sof/intel/cnl.c

```diff
@@ -252,6 +252,7 @@ const struct sof_intel_dsp_desc cnl_chip_info = {
 	.ipc_ack = CNL_DSP_REG_HIPCIDA,
 	.ipc_ack_mask = CNL_DSP_REG_HIPCIDA_DONE,
 	.ipc_ctl = CNL_DSP_REG_HIPCCTL,
-	.rom_init_timeout	= 300,
+	.rom_init_timeout = 300,
+	.dmic_num = 2,
```

Comment:
what does this represent? the number of PDM interfaces, the number of dailinks, the number of actual microphones? The latter can not possibly come from fixed descriptors but would need to come from topology or NHLT.

Reply:
It is the number of dai links for dmic supported by SOF driver and cAVS driver.
````
They are defined in 
struct snd_soc_dai_driver skl_dai[] or static struct snd_soc_dai_driver skl_platform_dai[]
= {
...
{
	.name = "DMIC01 Pin",
},
{
	.name = "DMIC16k Pin",
},
````